packages/plugin-core/src/fileWatcher.ts

```diff
@@ -91,6 +92,11 @@ export class VaultWatcher {
     note = NoteUtils.hydrate({ noteRaw: note, noteHydrated });
     const links = LinkUtils.findLinks({ note, engine: eclient });
     note.links = links;
+    const anchors = AnchorUtils.findAnchors(
```

Comment:
nitpick on style: Promises are usually awaited on at the call site or chained with a `then` function.